local/bin/scheme-live

```diff
@@ -1,6 +1,31 @@
-#!/bin/sh
+#!/bin/bash
 
-set -xe
+# set -x # echo debug line by line
+set -e # Exit on error
 
-echo "Running checks..."
-echo "Arguments are:" $@
+# SHORT_DESCRIPTION="My command with various sub commands!"
+# HELP_MAIN="usage: $(basename "$0") <sub-commands> [options]
+#
+# sub-commands are made from alias, function, and executables in PATH."
+
+# scheme-live-chibi () {
+#     echo "chibi install in progress..."
+
+#     source $SCHEME_LIVE_PREFIX/src/shell-subcommand.sh
+# }
+
+alias scheme-live-chibi=scheme-live-chibi
+
+scheme-live-version () {
+    # Based on https://srfi.schemers.org/srfi-176/
+    echo '(command "scheme-live")'
+    echo '(website "https://github.com/scheme-live/live/")'
+    echo '(languages scheme r6rs r7rs)'
+    echo "(install-dir \"$SCHEME_LIVE_PREFIX\")"
+    echo '(scheme.id live)'
+    echo '(release "2021.0")'
+    echo '(release.name "Fantastic Moon")'
+    echo "(build.date \"$(date --iso=seconds)\")"
```

Comment:
`date -u '+%Y-%m-%dT%H:%MZ'` is portable to non-GNU `date`, or `date '+%Y-%m-%dT%H:%M%z' | sed 's@^.\{19\}@&:@'` if you want local time instead of GMT